### PR TITLE
Grocy docker instance works on release

### PR DESCRIPTION
**Pull request by BlueBlueBlob**
## Description

Docker Grocy instance will run release version and not pre-release.

**Related issue (if applicable):** fixes #<issue number goes here>

## Checklist
  - [x] The code change is tested and works locally.
  - [x] tests pass. **Your PR won't be merged unless tests pass**
  - [x] There is no commented out code in this PR

### CODE REVIEW

**Comment by SebRut** [comment]
LGTM

**Comment by SebRut** [comment]
Do we need to change stuff in the testing section of README.md for this?

**Comment by BlueBlueBlob** [comment]
> Do we need to change stuff in the testing section of README.md for this?

No. It's only Travis stuff.

File: README.md [hunk 1]
```diff
@@ -36,19 +36,13 @@ for entry in grocy.stock():
 You need a Grocy instance running in demo mode at localhost with https (docker or a php server)
 You can setup url, port and ssl in test/test_const.py
 Easy case : install docker and docker-compose and run :
```
Comment (SebRut) [inline README.md]: I think you need to add an actual new line for the line break to work. So free line before line 38 and line 39.

**Comment by BlueBlueBlob** [comment]
Sorry, done.

File: README.md [hunk 2]
```diff
@@ -36,19 +36,14 @@ for entry in grocy.stock():
...
```
Comment (SebRut) [inline README.md]: Could you remove `Easy case : ` and add an empty line before it

**Pull request merged**